### PR TITLE
Add LACP fast rate option to Interfaces HLD

### PR DESCRIPTION
**Pull request by ArthiSivanantham**

### CODE REVIEW

File: doc/mgmt/SONiC_OC_Interface_HLD.md [hunk 1]
```diff
@@ -383,7 +387,7 @@ sonic(conf-if-po4)# no switchport access Vlan
 ```
 #### 3.6.2.1.2 PORTCHANNEL
 #### Create a PortChannel
-`interface PortChannel <channel-number> [mode <active | on>] [ min-links <value> ] [fallback] `<br>
```
Comment (nnag-broadcom) [inline doc/mgmt/SONiC_OC_Interface_HLD.md:386]: there is typo , end of the command is <cr> , not <br>

File: doc/mgmt/SONiC_OC_Interface_HLD.md [hunk 2]
```diff
@@ -392,6 +396,7 @@ sonic(conf-if-po4)# no switchport access Vlan
    mode - active<br>
    min-links - 0<br>
    fallback - disabled<br>
+   fast_rate - disabled<br>
 ```
 sonic(config)# interface PortChannel 1 mode active min-links 2 fallback
```
Comment (nnag-broadcom) [inline doc/mgmt/SONiC_OC_Interface_HLD.md:401]: Once the user specifies Min-links and fall back , he cannot change the values unless he deletes the port-channel and recreates it. Is there any other way to change these values without deleting the port-channel?

**Review by nnag-broadcom — CHANGES_REQUESTED** [review]
.

File: doc/mgmt/SONiC_OC_Interface_HLD.md (hunk 2 above)
Comment (bhavini-gada) [reply doc/mgmt/SONiC_OC_Interface_HLD.md:401]: This behavior was requested by @ben-gale in 3.0, please discuss internally.

File: doc/mgmt/SONiC_OC_Interface_HLD.md [hunk 3]
```diff
@@ -392,6 +396,7 @@ sonic(conf-if-po4)# no switchport access Vlan
    mode - active<br>
    min-links - 0<br>
    fallback - disabled<br>
+   fast_rate - disabled<br>
```
Comment (anil-kolkaleleti) [inline doc/mgmt/SONiC_OC_Interface_HLD.md:399]: Fast_rate is to configure the rate at which control packets are sent by the Link Aggregation Control Protocol (LACP) ??
If we enable the fast_rate config, what is the default rate at which lacp PDUs will be sent?
What are the configurable rates?
is there a no form of this command ??

Comment (joyas-joseph) [reply doc/mgmt/SONiC_OC_Interface_HLD.md:399]: LACP has only 2 rates - slow (once in 30 seconds) and fast (once in 1 sec).

This is a parameter that cannot be changed once the portchannel is created. So, the only way to change it is to delete and recreate the portchannel.

**Comment by ArthiSivanantham** [comment]
> what is fast_rate? Could not find the description anywhere.
> Is it short and long timeout of LACP?
> is fast_rate (i.e. short timeout) is disabled by default?


-- It's the interval/period between LACP messages. As mentioned by Joyas, LACP has 2 rates - FAST and SLOW (default).

And yes, "fast_rate" is disabled by default.

**Pull request merged**